eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Proposals.hs

```diff
@@ -0,0 +1,88 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Test.Cardano.Ledger.Conway.Proposals where
+
+import Cardano.Ledger.Conway
+import Cardano.Ledger.Conway.Governance
+import Control.DeepSeq (force)
+import Control.Exception (evaluate)
+import Data.Either (isRight)
+import Data.Foldable (foldl', toList)
+import qualified Data.Map.Strict as Map
+import Data.Maybe (fromMaybe)
+import Data.Sequence (fromList)
+import qualified Data.Sequence as Seq
+import Data.Sequence.Strict (StrictSeq)
+import qualified Data.Set as Set
+import Test.Cardano.Ledger.Common
+import Test.Cardano.Ledger.Conway.Arbitrary (
+  ProposalsForEnactment (..),
+  ProposalsNewActions (..),
+ )
+
+takeTail :: StrictSeq (GovActionId c) -> [GovActionId c]
+takeTail = reverse . take 4 . reverse . toList
```

Comment:
Can there be a better way? :sweat_smile:

Reply:
Sure:
```suggestion
takeTail n = toList . StrictSeq.takeLast n
```